src/Sylius/Behat/Context/Setup/ProductReviewContext.php

```diff
@@ -62,30 +63,52 @@ public function thisProductHasAReviewTitledAndRatedAddedByCustomer(
     }
 
     /**
-     * @Given /^(this product) has(?:| also) a review titled "([^"]+)" and rated (\d+) with a comment "([^"]+)" added by (customer "[^"]+")$/
+     * @Given /^(this product) has(?:| also) a rejected review titled "([^"]+)" and rated (\d+) added by (customer "[^"]+")(?:|, created (\d+) days ago)$/
      */
-    public function thisProductHasAReviewTitledAndRatedWithACommentAddedByCustomer(
+    public function thisProductHasARejectedReviewTitledAndRatedAddedByCustomer(
         ProductInterface $product,
-        $title,
-        $rating,
-        $comment,
+        string $title,
+        int $rating,
         CustomerInterface $customer,
-    ) {
-        $review = $this->createProductReview($product, $title, $rating, $comment, $customer);
+        ?int $daysSinceCreation = null,
+    ): void {
+        $review = $this->createProductReview($product, $title, $rating, $title, $customer, ProductReviewTransitions::TRANSITION_REJECT);
+        if (null !== $daysSinceCreation) {
+            $review->setCreatedAt(new \DateTime('-' . $daysSinceCreation . ' days'));
+        }
 
         $this->productReviewRepository->add($review);
     }
 
     /**
-     * @Given /^(this product) has(?:| also) a new review titled "([^"]+)" and rated (\d+) added by (customer "[^"]+")$/
+     * @Given /^(this product) has(?:| also) a new review titled "([^"]+)" and rated (\d+) added by (customer "[^"]+")(?:|, created (\d+) days ago)$/
      */
-    public function thisProductHasAReviewTitledAndRatedAddedByCustomerWhichIsNotAcceptedYet(
+    public function thisProductHasANewReviewTitledAndRatedAddedByCustomer(
         ProductInterface $product,
-        $title,
-        $rating,
+        string $title,
+        int $rating,
         CustomerInterface $customer,
-    ) {
+        ?int $daysSinceCreation = null,
+    ): void {
         $review = $this->createProductReview($product, $title, $rating, $title, $customer, null);
+        if (null !== $daysSinceCreation) {
+            $review->setCreatedAt(new \DateTime('-' . $daysSinceCreation . ' days'));
+        }
```

Comment:
```suggestion

        if (null !== $daysSinceCreation) {
            $review->setCreatedAt(new \DateTime('-' . $daysSinceCreation . ' days'));
        }
```

Reply:
Fixed in https://github.com/Sylius/Sylius/pull/16325